docs/services/navigation/_index.md

```diff
@@ -554,3 +554,119 @@ obstacles = await my_nav.get_obstacles()
 
 {{% /tab %}}
 {{< /tabs >}}
+
+## Concepts
+
+### CompassHeading
+
+Compass heading measurements are read by the following {{< glossary_tooltip term_id="model" text="models" >}} of [movement sensor](/components/movement-sensor/):
+
+- [gps-nmea](/components/movement-sensor/gps/gps-nmea/)
+- [gps-nmea-rtk-pmtk](/components/movement-sensor/gps/gps-nmea-rtk-pmtk/)
+- [gps-nmea-rtk-serial](/components/movement-sensor/gps/gps-nmea-rtk-serial/)
+
+An example of a `Compass Heading` reading:
+
+``` go
+gps.GetCompassHeading{175}
+```
+
+Compass heading readings are taken through the movement sensor API's [`GetCompassHeading()`](/components/movement-sensor/#getcompassheading).
+If you want to read compass headings, [configure a capable movement sensor](/components/movement-sensor/#configuration) on your robot.
+Use compass heading readings to determine the *bearing* of your robot, or, the [cardinal direction](https://en.wikipedia.org/wiki/Cardinal_direction) that your robot is facing.
+
+### Orientation
+
+Orientation measurements are read by the following {{< glossary_tooltip term_id="model" text="models" >}} of [movement sensor](/components/movement-sensor/):
+
+- [imu-wit](/components/movement-sensor/imu/imu-wit/)
+- [imu-vectornav](/components/movement-sensor/imu/imu-vectornav/)
+
+An `Orientation` reading specifies the orientation of an object in 3D space as an "orientation vector", or, its position within the [cartesian coordinate system](https://en.wikipedia.org/wiki/Cartesian_coordinate_system) relative to some specific `origin` point that you, the user, need to choose and configure for your robot in Viam's [frame system](/services/frame-system/).
+
+An example of an `Orientation` reading:
+
+``` golang
+imuwit.GetOrientation{o_x: -0.58, o_y: -0.37, o:z 0.72, theta: 60.93}
```

Comment:
```suggestion
orientation, err := imuwit.Orientation(context.Background, nil)
```

orientation is of the form o_x: -0.58, o_y: -0.37, o:z 0.72, theta: 60.93,
same for all other APIs.